conda-recipe/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: csxtools
+  version: {{ environ.get('GIT_DESCRIBE_TAG', 'GIT_STUB') }}.post{{ environ.get('GIT_DESCRIBE_NUMBER','') }}
```

Comment:
probably need to strip the first letter? not sure what tagging convention stuart is using.